recipes/datatable/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "datatable" %}
+{% set version = "0.11.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datatable-{{ version }}.tar.gz
+  sha256: 19c602711e00f72e9ae296d8fa742d46da037c2d3a2d254bdf68f817a8da76bb
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
+  ignore_run_exports:
+    - python  # we want a specific python version
+    # fix overlinking/overdependency error
+    - libgcc  # [linux]
+    - libcxx  # [osx]
+
+# numpy and pandas are _optional_ dependencies of dataframe
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+  run:
+    - python
+    # fix overlinking error
+    - libgcc   # [linux]
+    - libcxx   # [osx]
```

Comment:
Mixing compilers and compiler versions is very dangerous - it should be fine to remove this part as well.

Reply:
The build failed without this. If you check the CI output for 839392a you can see this. Is there a different way to solve this?

Reply:
That one failed because you still had `noarch: python` in the recipe https://github.com/jeromebaum/staged-recipes/blob/f24ea90f1f0f5180b3b9be426937b1e514a7c1f8/recipes/datatable/meta.yaml#L15

Reply:
I'll try removing this and see where things are. (also the other changes)

Is there a place somewhere where I can find the full invocation of conda-build that reproduces the CI checks? I found `--error-overlinking` but I don't think I have a full replicate of the CI.